browser/BUILD.gn

```diff
@@ -151,7 +150,6 @@ source_set("browser_process") {
     "//brave/components/decentralized_dns/buildflags",
     "//brave/components/gemini/browser/buildflags",
     "//brave/components/greaselion/browser/buildflags",
-    "//brave/components/ipfs/buildflags",
```

Comment:
I think this one (and the import for the buildflags.gni file above) shouldn't be moved from here into ipfs/sources.gni because this is actually required by sources that are still listed from this BUILD.gn file i.e.:
  * brave/browser/brave_browser_process_impl.h
  * brave/browser/brave_content_browser_client.cc
  * brave/browser/brave_profile_prefs.cc
  * brave/browser/brave_tab_helpers.cc
  * [...] (and others)

Additionally, there are no includes for `brave/components/ipfs/buildflags/buildflags.h` from //brave/browser/ipfs/*, so it looks like this should stay in here instead.